server/endpoint/health/searcher/testdata/case-0-aws.golden

```diff
@@ -1,116 +1,188 @@
 {
-    "cluster": {
-        "health": "green",
-        "state": "normal"
+  "cluster": {
```

Comment:
Why the indentation here changed?

Reply:
There's no real standard for JSON, but 2 spaces of indentation is the de facto standard for JS so I went with that.